shelley/design-spec/delegation_design_spec.tex

```diff
@@ -2718,32 +2747,108 @@ \subsection{Reward Splitting}
 
 These calculations proceed in two steps: First, rewards are split
 amongst \emph{pools}. Next, each pool splits its share of \(R\) amongst
-its leader and its members.
+its operator and its members.
+
+\subsubsection{Stake, Performance, and Block Production}
+\label{stake-performance-and-block-production}
+
+The incentives scheme developed in~\citep{bkks2018} distributes rewards solely
+on the basis of the size of pools in terms of the stake that they control. This
+does eliminate any incentive for a pool to try to sabotage another pool (be it
+through ignoring their blocks, or even by mounting DoS attacks on their nodes).
+However, it does not incentivise pool operators to ensure that their pool
+performs well (i.e., produces most of the blocks it is eligible to create). This
+invites freeriders, pools that rely on other pools to maintain the system, and
+collect rewards without doing any work.
+
+For Cardano, we will stay close to the scheme of~\citep{bkks2018}, but we will
+also take into account the \emph{performance} of a pool. Unfortunately, tracking
+the performance of a pool is not trivial in protocols without a public leader
+schedule, like Ouroboros Praos.
+
+In order to get a handle on the performance of a pool, let us consider the
+fraction \(\beta\) of all blocks within a given epoch that the pool created.
+
+There are multiple factors that influence \(\beta\): the stake of the pool, how
+well it performs, and some randomness due to the leader election and the overall
+performance of the other pools and the network. We can write\footnote{We assume
+  a linear relation between stake and the number of blocks that a pool is
+  eligible to create. Strictly speaking, this is not the case for Ouroboros
+  Praos. However, the error when linearising the leader election function is
+  small, particularly for the range of parameters we are considering for
+  Cardano.}
+\[\beta = \sigma p r_e \,,\]
+where
+\begin{enumerate}
+\item[\(\sigma\)] is the relative stake \(\sigma\) of the pool (the fraction of
+  the absolute stake that the pool controls),
+\item[\(p\)] is the performance \(p\) of the pool, i.e. the fraction
+  \[
+  p = \frac{n}{\max(N,1)}
+  \]
+  of the number \(n\) blocks it successfully added to the chain and the number
+  \(N\) of slots it was elected as a leader, and
+\item[\(r_e\)] is a factor that captures the relation between the relative stake
+  \(\sigma\) of the pool, the number \(N\) of slots it is elected as a leader,
+  and the total number \(\overline{N}\) of blocks that were added to the chain during
+  the epoch. To be precise, we have
+  \[r_e = \frac{N}{\sigma \overline{N}}\,.\]
```

Comment:
Strictly speaking, this fails if \overline{N} is zero. But I guess if we ever encounter an epoch that doesn't produce a single block, we have bigger problems anyway?

Reply:
Yes, I guess :)

But doing the `max(1, N)` thing can't hurt.

Reply:
We don't want to add insult to injury and divide by zero if the world has ended.